README.md

```diff
@@ -15,3 +15,52 @@ This repository houses the open-source components of Mattermost Private Cloud. O
 ## Get Involved
 
 - [Join the discussion on Private Cloud](https://community.mattermost.com/core/channels/cloud)
+
+## Developing
+
+### Environment Setup
+
+1. Install [Go](https://golang.org/doc/install)
+2. Install [Terraform](https://learn.hashicorp.com/terraform/getting-started/install.html) version v0.11.14
+3. Install [kops](https://github.com/kubernetes/kops/blob/master/docs/install.md) version 1.11.1
```

Comment:
I think we are currently using the latest 1.12.X kops release. I currently have `Version 1.12.2` installed.